alg/gdaltransformer.cpp

```diff
@@ -437,15 +437,17 @@ GDALSuggestedWarpOutput2( GDALDatasetH hSrcDS,
         static_cast<double>(std::min(nInYSize, nInXSize)) / N_PIXELSTEP + 0.5);
     if( nSteps < 20 )
         nSteps = 20;
-    nSteps = std::min(nSteps, 100);
+    else if( nSteps > 100 )
+        nSteps = 100;
+    int nStepsPlusOne = nSteps + 1;
```

Comment:
this should be moved after the "retry:" label as nSteps can be modified at line 469 before a goto retry